### DISCUSSION TITLE
Changing style of scrollbar

### DISCUSSION THREAD

**Discussion by jzhang172** [idiotWu/smooth-scrollbar#532] [Q&A]
I tried adding but did not work:
```
.scrollbar-track {
  background: none !important;
}
.scrollbar-thumb {
  background: rgb(151 151 151 / 50%) !important;
}

```

https://stackblitz.com/edit/angular-ivy-kvauhk?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.css

**Comment by sadeghbarati** [comment]
I sense and understand the Lack of documentation here, but check README section more for now

https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbardetachstyle

**Reply by jzhang172** [reply]
Okay, thanks for the link!
